pkg/asset/agent/installconfig_test.go

```diff
@@ -351,7 +351,7 @@ platform:
 pullSecret: "{\"auths\":{\"example.com\":{\"auth\":\"authorization value\"}}}"
 `,
 			expectedFound: false,
-			expectedError: "invalid install-config configuration: [ControlPlane.Architecture: Unsupported value: \"s390x\": supported values: \"amd64\", \"arm64\", \"ppc64le\", Compute[0].Architecture: Unsupported value: \"s390x\": supported values: \"amd64\", \"arm64\", \"ppc64le\"]",
+			expectedError: "invalid install-config configuration: [ControlPlane.Architecture: supported values: \"amd64\", \"arm64\", \"ppc64le\", \"s390x\", Compute[0].Architecture: Unsupported value: \"s390x\": supported values: \"amd64\", \"arm64\", \"ppc64le\"]",
```

Comment:
An additional change is needed in expectedError. s390x should be moved to the supported values list under Compute[0].Architecture.

Also, the test case assumes s390x is an unsupported architecture. Since this patch makes it a supported one, we'll need to set something else here as unsupported. 
Change
`architecture: s390x`
to
`architecture: unsupportedArch`

Reply:
Changed it, it was an unseen mistake

Reply:
Removed the SNO invalid arch test case as now s390x supports SNO